client/js/components/map/admin/MapAdmin.vue

```diff
@@ -82,42 +82,49 @@
       }" />
 
     <div class="layout__item u-1-of-1 text-right u-mt-0_5 space-inline-s">
-      <input
-        class="btn btn--primary"
+      <dp-button
+        data-cy="save"
         :disabled="!areScalesSuitable"
-        type="submit"
         name="saveConfig"
-        :value="Translator.trans('save')"
-        @click="() => save()">
-      <input
+        :text="Translator.trans('save')"
+        type="submit"
+        @click="() => save()" />
+      <dp-button
         v-if="hasPermission('area_admin_single_document')"
-        class="btn btn--primary"
+        data-cy="saveAndReturn"
         :disabled="!areScalesSuitable"
-        type="submit"
         name="submit_item_return_button"
-        :value="Translator.trans('save.and.return.to.list')"
-        @click="() => save(true)">
-      <a
-        class="btn btn--secondary"
-        :href="Routing.generate('DemosPlan_element_administration', { procedure: procedureId })">
-        {{ Translator.trans('abort') }}
-      </a>
+        :text="Translator.trans('save.and.return.to.list')"
+        type="submit"
+        @click="() => save(true)" />
+      <dp-button
+        color="secondary"
+        data-cy="abort"
+        :href="Routing.generate('DemosPlan_element_administration', { procedure: procedureId })"
+        :text="Translator.trans('abort')" />
+      <dp-button
+        color="secondary"
+        data-cy="reset"
+        :text="Translator.trans('reset')"
+        variant="outline"
+        @click="reset" />
```

Comment:
I find it somewhat unusual to find both buttons next to each other. Usually i find either "reset", or "abort".

So we got those...
- "Speichern"
- "Speichern und zurück" (only in projects where there is a location to navigate back to)
- "Abbrechen" (which should also only show up in projects where there is a location to navigate back to, but it is currently always shown)
- "Zurücksetzen" (which has the same effect on data entry as "Abbrechen" but stays on the same page)

I'd suggest to offer this combination of buttons for projects "with an overview page" (leaving out the "zurücksetzen" because users can afford nearly the same thing with "abbrechen"):
**[Speichern] [Speichern und zurück] [Abbrechen]**

And for projects without the overview we are left with
**[Speichern] [Zurücksetzen]**